css-ecommerce-page/index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ecommerce Challenge</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap"
+      rel="stylesheet" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="page-container">
+      <div class="row">
+        <h2>New Arrivals</h2>
+      </div>
+
+      <div class="row">
+        <div class="content-container">
+          <!--Product 1-->
+          <div class="five-columns">
+            <section>
```

Comment:
These should be swapped. The `section` should be the parent, and the `div` elements should replace your `section`s. `section` means all the content inside is related. So since the price / title / image are all related, they should all be inside a `section`.

Reply:
Ah okay, that makes more sense haha. Thanks for flagging - I made the adjustments! 

![Screenshot 2023-05-25 at 7 14 13 PM](https://github.com/j-alejandro-araujo/c0523-code-solutions/assets/91649194/e2ceb49c-ff06-4a61-94d6-20bce5092951)

Reply:
![Screenshot 2023-05-25 at 7 20 32 PM](https://github.com/j-alejandro-araujo/c0523-code-solutions/assets/91649194/572f755e-8489-4d60-8894-4fbddc2d285d)

I got a non-fast-forward error message when trying to push the changes on this branch. Any ideas how to resolve?

Github Docs advises the solution bellow - let me know your thoughts. I don't want to try anything before discussing w/ you first. 

![Screenshot 2023-05-25 at 7 28 28 PM](https://github.com/j-alejandro-araujo/c0523-code-solutions/assets/91649194/688705d6-c25b-48ef-b8ec-3e8549190dfe)